### DISCUSSION TITLE
2023-07-28

### DISCUSSION THREAD

**Discussion by nlom0218** [woowacourse-teams/2023-haru-study#197] [daily_meeting]
### 개인 일정
마코 - 2-3시 테코톡
노아 - 4-5시 근로

### 오늘의 할 일

- 정식 론칭 서비스 배포 완료(1.0 배포!)

#### 백엔드

- 재접속 안되는 문제 => 이슈파서 해결하기 서버 로직 문제 (백엔드는 해결됨, 프론트에 수정사항 전달하기)

#### 프론트엔드 리팩터링 목록

- [ ] 도메인 용어 및 문장 상수화
- [x] 컴포넌트 폴더 구조
- [x] 커스텀 훅 폴더 구조
- [x] 라우터 및 api url 상수화
- [ ] fetch
- [ ] 전체적인 코드 정리

#### 다같이
- 소켓, 폴링 양방향 통신 어떻게 할지

#### 백엔드
- A라는 스터디를 참여후 B스터디 참여한 뒤 다시 A스터디에 참여할 시 닉네임을 찾지 못한 이슈(로그인이 된다면 해결될 문제이긴 함)
- 쿠키 관리 정책 재논의

### 언젠가 할 일

- 기획 변경된 이유 정리
- 질문 PlaceHolder로 유의미한 응답 예시를 제공하는 기능 (목표설정 가이드라인)
- 브랜치 전략 결정하기(dev -> main)

**Comment by nlom0218** [comment]
### 체크인

9점 - 벌써... 금요일ㅠㅠㅠ -1

### 체크아웃

8점 - 오늘 처음으로 제가 인생을 잘 살고 있는지 깊은 의문이 들었습니다. (잠실에 대한 정이 조금씩 떨어지는...)

**Comment by woosung1223** [comment]
### 체크인
7점 - 오늘 엽토가 은근 일찍와서 -3점

### 체크아웃
10점 - 금요일이기도 하고 마코가 좋은 의견 내줘서 10점!

**Comment by yeopto** [comment]
### 체크인

6점 - 많이 피곤하네요

### 체크아웃

10점 - 뭔가 한 주 열심히 달려온 것 같아서 뿌듯해서 +4

**Comment by jaehee329** [comment]
### 체크인
8점 - 드디어 금요일이라 어제보다 점수가 올랐습니다

### 체크아웃
10점 - 마지막에 대화 좋았습니다. 그리고 주말이잖아요

**Comment by aak2075** [comment]
### 체크인

10점 - 불금, 칼퇴, 회식 삼위일체

### 체크아웃

10점 - 값진 토론 즐거웠습니다

**Comment by woo-jk** [comment]
### 체크인
9점 - 어제 일찍자서

### 체크아웃
9점 - 나쁘지 않아서

**Comment by MoonJeWoong** [comment]
### 체크인
10점 - 기본적인 서비스 기능 배포에 성공한데다 어제 인프라 젠킨스 구축도 완료해보고 불금이라 기분이 너무 좋네요 ^^

### 체크아웃
9점 - 열심히 논의한 것 같은데 이걸 잘 정리해두지 못한 것 같아서 아쉽네요...
